e2e-tests/tfm/cypress/e2e/journeys/case-amendments/show-amendment-button/deal-cancellation-enabled/show-amendment-button.spec.js

```diff
@@ -0,0 +1,44 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../../../fixtures/deal-AIN';
+import { PIM_USER_1, BANK1_MAKER1, ADMIN } from '../../../../../../../e2e-fixtures';
+import { submitDealCancellation } from '../../../../../support/trade-finance-manager-ui/submit-deal-cancellation';
+
+context('Amendments page', () => {
+  describe('A cancelled deal', () => {
+    let dealId;
+    const dealFacilities = [];
+
+    before(() => {
+      cy.insertOneDeal(MOCK_DEAL_AIN, BANK1_MAKER1).then((insertedDeal) => {
+        dealId = insertedDeal._id;
+
+        const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+
+        cy.createFacilities(dealId, mockFacilities, BANK1_MAKER1).then((createdFacilities) => {
+          dealFacilities.push(...createdFacilities);
+        });
+
+        cy.submitDeal(dealId, dealType, PIM_USER_1);
```

Comment:
I dont think a PIM user can submit a deal

Reply:
Ive just realised that this is directly calling the tfm-api endpoint, which is normally called by portal-api

Reply:
it _shouldnt_ need a tfm user to call this, Im not sure why its doing this